ProcessAPI/ComputeEngine.java

```diff
@@ -0,0 +1,21 @@
+
+public class ComputeEngine implements ProcessInterface, ProcessResponse{

```

Comment:
It's a little unusual to implement multiple APIs with one class; typically that only happens with very general 'supports some operation' type interfaces (Closeable, Identifiable), where by implementing the interface the class is saying it has some sort of adjective-related property rather than a type property (if that makes sense). So a class generally wouldn't implement both Animal and Zoo (both 'type of data'/'collection of behavior' interfaces), but a Goat class might implement Hoofed, Horned, and EscapeArtist as interfaces (all 'adjective' style interfaces). Adjective-style interfaces are much less common that type-of-data interfaces, and tend to only show up in fairly specialized situations - in this case, I'd just make a new ProcessResponseImpl class.